### PR TITLE
Fixes due to css-changes on top.gg

### PR DESCRIPTION
**Pull request by Hopefuls**
This fix includes the following sections that were partially to fully nonfunctional due to the recent css-changes on top.gg:

- DSL (top.gg servers)
- DBL (top.gg bots)
- top.gg Staff Sections (including queue etc)

### CODE REVIEW

File: websites/T/top.gg/dist/metadata.json [hunk 1]
```diff
@@ -16,7 +20,7 @@
     "de": "Rich Presence für top.gg - Die größte Discord Bot Liste und mehr"
   },
   "url": "top.gg",
-  "version": "1.3.7",
+  "version": "1.4.0",
```
Comment (Bas950) [inline websites/T/top.gg/dist/metadata.json:23]: Not a big change, v1.3.8

File: websites/T/top.gg/dist/metadata.json [hunk 2]
```diff
@@ -7,6 +7,10 @@
     {
       "name": "Eren",
       "id": "478937019758673931"
+    },
+    {
+      "name": "Aurel",
+      "id": "669452973755072524"
     }
```
Comment (Bas950) [inline websites/T/top.gg/dist/metadata.json:14]: Not a big change, dont add yourself as contributor

**Review by Bas950 — CHANGES_REQUESTED** [review]
Upload proof of it working

**Comment by Hopefuls** [comment]
> Upload proof of it working

Please bear in mind that the staff-section is not viewable by the public and confidential, thus only being able to screenshot the actual presence, not the site. Thanks for your understanding.

Main Pages
Bots: https://i.imgur.com/mkzaTFD.png
Servers: https://i.imgur.com/D53q8pN.png


Bots 

Tag Examples
Music Bots: https://i.imgur.com/KGEFkk7.png
Top Bots: https://i.imgur.com/44oevcE.png

Bot Page Example
https://i.imgur.com/eXurMjE.png 

User Profile
https://i.imgur.com/GPdJ6KQ.png (previously broken)


DSL (top.gg servers)

Tag Examples
Gaming Discord Servers: https://i.imgur.com/105k25Y.png (previously broken)
Top Discord Servers: https://i.imgur.com/B4IBx11.png (previously broken)

Server Page Example
https://i.imgur.com/cDstBx8.png (previously broken)

User's Servers (partially censored due to private servers etc)
https://i.imgur.com/v9J181v.png

Staff Sections (only able to show the Presence, as this section is restricted to staff and confidential)

Main Moderation Section: https://i.imgur.com/vIxdEog.png
Verification Queue: https://i.imgur.com/wxmpYtA.png
Viewing an unapproved Bot: https://i.imgur.com/fCxnUwp.png
Declining a Bot: https://i.imgur.com/s9OwVb0.png (previously broken)

**Review by Bas950 — APPROVED** [review]
Upload proof of it working

**Review by ririxidev — APPROVED** [review]

**Pull request merged**